.gitignore

```diff
@@ -8,7 +8,6 @@ node_modules/
 /.psa*
 *.swp
 *#*
-package-lock.json
```

Comment:
why are we including this? adaik it is not standard practice in purescript libraries.

Reply:
why are we ignoring `package-lock.json` You mean?